src/components/CJobs.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="px-4 pt-2">
+    <div class="w-full">
+      <div class="flex float-right w-48 mb-3 search">
+        <input
+          class="placeholder-gray-700 bg-gray-100 mr-2 text-sm font-medium leading-5  focus:outline-none  py-2 px-2 block w-full appearance-none leading-normal"
+          id="filter"
+          placeholder="Search..."
+          type="text"
+          v-model="filter"
+        />
+        <div class="loader" v-if="isLoading">Loading...</div>
+      </div>
+    </div>
+    <table class="w-full">
+      <thead>
+        <tr class="bg-gray-100 border-4 h-8">
+          <c-th
+            v-for="(column, index) in columns"
+            :label="column.label"
+            :name="column.name"
+            :key="index"
+            :isSortable="column.sortable"
+            :sortedColumn="sortedColumn"
+            @onClickSort="setSortColumnAndDirection"
+          ></c-th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(job, index) in displayedJobs" :key="index">
+          <td class="border px-4 py-2" v-for="column in columns" :key="column.name">
+            <pre class="text-xs whitespace-normal ml-2 bg-white"> {{ job[column.name] }}</pre>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+<script>
+import { mapState } from 'vuex';
+import CTh from '@/components/CTh';
+export default {
+  name: 'CJobs',
+  components: { CTh },
+
+  data() {
+    return {
+      filter: '',
+      columns: [
+        { label: 'Actor Name', name: 'actorName', sortable: true },
+        { label: 'Daily Time', name: 'dailyTime' },
+        { label: 'Enabled', name: 'enabled', sortable: true },
+        { label: 'Interval', name: 'interval' },
+        { label: 'ISO Weekday', name: 'isoWeekday' },
+        { label: 'Args', name: 'args' },
+        { label: 'Kwargs', name: 'kwargs' },
+        { label: 'Last Queued', name: 'lastQueued', sortable: true },
+        { label: 'Time Zone', name: 'tz' }
+      ],
+      sortedColumn: null,
+      sortDirection: 'asc'
+    };
+  },
+  computed: {
+    ...mapState(['jobs', 'isLoading']),
+
+    filteredJobs() {
+      if (!this.filter) {
```

Comment:
this is duplicated code: define a function filterTable in an utils file and use it here. defineition should be 
`filterTableData(data, filter, filterKeys) => fiteredData`
same with below add a function `sortTableData(filteredData, sortDirection, sortedColumn)`
finally
`[this.sortedColumn, this.sortDirection] = setSortColumnAndDirection(columnName, this.sortedColumn, this.sortDirection)`